### PR TITLE
Add gitops structure documentation

### PR DESCRIPTION
**Pull request by hanzala1234**

### CODE REVIEW

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/c8d43a0ef9f7016d0e7e47fa0aabe749f992a19f/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/bd5a576296588e440b567de8d177f336b75f58b8/checks) has Failed!

File: content/sre/gitops/structure.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+```.
+├── 01-tenant
+│   ├── 01-app
+│   │   ├── 00-env
+│   │   │   └── nginx.yaml
```
Comment (rasheedamir) [inline content/sre/gitops/structure.md]: what is this nginx.yaml?

File: content/sre/gitops/structure.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+│   │   │   └── values.yaml
+│   │   ├── 02-env
+│   │   │   ├── Chart.yaml
+│   │   │   └── values.yaml
+│   │   └── 03-env
+│   │       ├── Chart.yaml
+│   │       └── values.yaml
+│   ├── 02-app
+│   │   ├── 00-env
+│   │   │   └── argocd.yaml
```
Comment (rasheedamir) [inline content/sre/gitops/structure.md]: what is only argocd.yaml?

File: content/sre/gitops/structure.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+│   │   │   └── values.yaml
+│   │   ├── 02-env
+│   │   │   ├── Chart.yaml
+│   │   │   └── values.yaml
+│   │   └── 03-env
+│   │       ├── Chart.yaml
+│   │       └── values.yaml
+│   └── configs
+│       ├── 00-env
+│       │   └── space.yaml
```
Comment (rasheedamir) [inline content/sre/gitops/structure.md:29]: what 00-env has no argocd folder?

File: content/sre/gitops/structure.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+│       │   └── tenants.yaml
+│       └── tenants
+│           ├── 01-tenant.yaml
+│           ├── quota.yaml
+│           └── 02-tenant.yaml
+└── README.md
+
+```
+
+The same configured gitops directory can be found [here](https://github.com/stakater/gitops-config-template)
```
Comment (rasheedamir) [inline content/sre/gitops/structure.md:115]: point them to the right branch

File: content/sre/gitops/structure.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+├── 03-sre
+│   ├── 01-cluster
+│   │   ├── argocd
+│   │   │   ├── 00-env
+│   │   │   │   └── 01-tenant.yaml
+│   │   │   ├── 01-env
+│   │   │   │   ├── 01-tenant.yaml
+│   │   │   │   └── 02-tenant.yaml
+│   │   │   └── tenants.yaml
+│   │   └── tenants
```
Comment (rasheedamir) [inline content/sre/gitops/structure.md]: lets call this something else so, we can have tenant config in one place

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/43f252bb62f7c02446a927a111683f8a111bfea4/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/5fbef7eb63bd12d98981c6ef2421e8d298ea1c07/checks) has Failed!

**Comment by stakater-user** [comment]
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/ce86097637e123bccfcdc9aa050b6ee11a3ca3cc/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/127b568f06835a8c0d869a516fee3c114a7328ad/checks) has Failed!

File: content/sre/gitops/structure.md (hunk 4 above)
Comment (hanzala1234) [reply content/sre/gitops/structure.md:115]: will merge it to main branch

File: content/sre/gitops/structure.md (hunk 5 above)
Comment (hanzala1234) [reply content/sre/gitops/structure.md]: lets discuss

**Comment by stakater-user** [comment]
@rasheedamir Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/58bf51ecb91123ca14b30ca9695e508e21a8bc48/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/1dc03d84dc02e0c0189f16c1a6c23704ee481446/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/0637a3d7e1c04a67fe74f6dfc57dfcd14af6eadf/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-154-0637a3d7`

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/a4e06b702b15d6db0631b785c59f14f36800c55a/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-154-b1d93205`

**Review by cuttingedge1109 — APPROVED** [review]

**Comment by stakater-user** [comment]
@hanzala1234 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.99`

**Pull request merged**